packages/ui-components/src/lib/__mocks__/mockTransactionStore.ts

```
@@ -28,42 +27,32 @@ export const mockTransactionStore: Partial<TransactionStore> & {
 
-	handleRemoveOrderTransaction: async () => {
-		mockTransactionWritable.update((state) => ({
-			...state,
-			status: TransactionStatus.SUCCESS,
-			message: 'Order removed successfully',
-			hash: '0x789'
-		}));
-		return Promise.resolve();
-	},
-
 	checkingWalletAllowance: (message: string = '') =>
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)